src/ExposureHistory/index.tsx

```diff
@@ -1,16 +1,27 @@
-import React, { FunctionComponent } from "react"
+import React, { FunctionComponent, useCallback } from "react"
 
 import { useExposureContext } from "../ExposureContext"
 import History from "./History"
 
 import { ExposureInfo, ExposureDatum } from "../exposure"
+import { useFocusEffect } from "@react-navigation/native"
```

Comment:
Thoughts on moving this import to line 2 for consistency with the other files?

Reply:
Done!